scripts/go-build-ldflags.sh

```diff
@@ -13,13 +13,15 @@ export GIT_VERSION=$(git describe --tags --abbrev=0 2>/dev/null || echo unknown)
 export BUILD_DATE=$(date '+%F-%T') # outputs something in this format 2017-08-21-18:58:45
 export BUILD_TS_UNIX=$(date '+%s') # second since epoch
 export BASE_PACKAGE=github.com/uber/cadence/common/metrics
+SERVER_VERSION="${SERVER_VERSION:-'development'}"
 
 if [ "$MODE" = "LDFLAG" ]; then
   LD_FLAGS="-X ${BASE_PACKAGE}.Revision=${GIT_REVISION} \
   -X ${BASE_PACKAGE}.Branch=${GIT_BRANCH}               \
   -X ${BASE_PACKAGE}.Version=${GIT_VERSION}             \
   -X ${BASE_PACKAGE}.BuildDate=${BUILD_DATE}            \
-  -X ${BASE_PACKAGE}.BuildTimeUnix=${BUILD_TS_UNIX}"
+  -X ${BASE_PACKAGE}.BuildTimeUnix=${BUILD_TS_UNIX}    \
+  -X main.Version=${SERVER_VERSION}"
```

Comment:
- @meiliang86 we have revision already in vars here, so no need to add it to the version.  though adding it to the output may be nice, yeah.
- @longquanzheng is SERVER_VERSION noticeably different than GIT_VERSION?  and why not put this in BASE_PACKAGE as well?

Reply:
good point. I didn't pay much attention. doing it.

Reply:
@meiliang86 okay I ended up append the revision  in the version output like you said, because there is no easy way to add such a field. 

@Groxx again, thanks for your comment.